lib/modules/dosomething/dosomething_northstar/dosomething_northstar.auth.inc

```diff
@@ -49,8 +50,14 @@ function dosomething_northstar_openid_authorize() {
 
     dosomething_northstar_set_authorization_action($_GET['action'], $_GET['node'], $params);
 
-    // @TODO: Rename parameter on the Northstar side to 'title'.
-    $url_options['query'] = ['destination' => $_GET['title']];
+    $url_options['query'] = [
+      'destination' => $campaign->title,
+      'options' => [
```

Comment:
Should these be nested under this `options` array? It looked like they all got merged to the same level in your [corresponding Gateway pull request](https://github.com/DoSomething/gateway/pull/77/files).